nix/nixos/cardano-explorer-service.nix

```diff
@@ -0,0 +1,106 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.cardano-explorer;
+  inherit (import ../.. { }) cardano-explorer;
+in {
+  config = mkIf cfg.enable (let stateDirBase = "/var/lib/";
+  in {
+    networking.firewall.allowedTCPPorts = [ 3001 8100 ];
+
+    users.groups.${cfg.group}.gid = 10017;
+
+    users.users.${cfg.user} = {
+      description = "cardano-explorer daemon user";
+      uid = 10017;
```

Comment:
does the uid/gid need to be static? it would have less chance of conflict if it was just dynamically assigned on first deploy